test/testkit/mocks/backend/deployment.go

```diff
@@ -12,7 +12,7 @@ import (
 
 const (
 	replicas           int32 = 1
-	otelCollectorImage       = "europe-docker.pkg.dev/kyma-project/prod/tpi/otel-collector:0.95.0-0eb4394f"
+	otelCollectorImage       = "europe-docker.pkg.dev/kyma-project/prod/tpi/otel-collector:v20240313-7cd3d3a3"
```

Comment:
What happened to the version number in the image name?

Reply:
fixed